index.js

```diff
@@ -13,6 +13,16 @@ CleanCSSMinifier.prototype.type = 'stylesheet';
 
 CleanCSSMinifier.prototype.optimize = function(data, path, callback) {
   var error, optimized;
+
+  try {
```

Comment:
Interesting, that is how it was done over there... well I guess it's fine to leave this way here then.